services/blockchain-app-registry/shared/metadata.js

```diff
@@ -167,6 +176,14 @@ const getBlockchainAppsMetadata = async (params) => {
 			const chainMetaString = await read(`${appPathInClonedRepo}/${config.FILENAME.APP_JSON}`);
 			const chainMeta = JSON.parse(chainMetaString);
 			chainMeta.isDefault = appMetadata.isDefault;
+
+			const [blockchainApp] = (await requestIndexer('blockchain.apps', { name: chainMeta.chainName })).data;
```

Comment:
Let's please use the `chainID` to query the on-chain information instead. `chainName` is metadata.
```suggestion
			const [blockchainApp] = (await requestIndexer('blockchain.apps', { chainID: chainMeta.chainID })).data;
```